File: validator/validator/src/main/java/data/Colors.java
package data;

public class Colors {
    String primary, secondary;

    public String colorsValid() {

        String sOutputString = "";//erzeugt leeren Diagnostikstring

        if (primary == null || secondary == null) {//sind die Farben beide da?
            sOutputString = sOutputString + " A Colour is missing!" + "\n";
        }


        if (primary.equals(secondary)) {//sind die Farben gleich?
            sOutputString = sOutputString + " Primary and Secondary colour cannot be the same!" + "\n";
        }

        return sOutputString;
    }
}

File: Client/Client/Assets/Scripts/MenuScripts/TeamConfig/Colors.cs
using System.Collections;
using System.Collections.Generic;

[System.Serializable]
public class Colors
{
    public string primary { get; set; }
    public string secondary { get; set; }

    public Colors(string p, string s)
    {
        primary = p;
        secondary = s;
    }
}
